data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 3/IceCap2.meta

```diff
@@ -1,3 +1,4 @@
 Sonic the Hedgehog 3 & Knuckles - Ice Cap Act 2
 0x03
 bgm
+IceCavern,ZorasDomain
```

Comment:
The location line should have at least one group from each level of specificity (exact, high, mid and low).